### PR TITLE
python2 minor fix

### PR DESCRIPTION
**Pull request by mirageoasis**
In this pr I think change was too radical
[link](https://github.com/julien-duponchelle/python-mysql-replication/pull/515)

so I only remove python2 compatibliity and did some minor fixes

### CODE REVIEW

File: pymysqlreplication/tests/benchmark.py [hunk 1]
```diff
@@ -25,12 +25,12 @@ def consume_events():
         only_events=[UpdateRowsEvent],
         only_tables=["test"],
     )
-    start = time.clock()
+    start = time.perf_counter()
```
Comment (sean-k1) [inline pymysqlreplication/tests/benchmark.py:28]: perf_counter method is it possible 3.3, 3.4, 3.5 and 3.6?

**Review by sean-k1 — COMMENTED** [review]
Everything looks good except for this
Can you check?

File: pymysqlreplication/tests/benchmark.py (hunk 1 above)
Comment (mirageoasis) [reply pymysqlreplication/tests/benchmark.py:28]: > python

https://docs.python.org/3/library/time.html#time.perf_counter

yes it says this method was introduced in python 3.3

File: pymysqlreplication/tests/base.py [hunk 2]
```diff
@@ -84,6 +84,13 @@ def isMySQL8014AndMore(self):
             return True
         return version == 8.0 and version_detail >= 14
 
+    def isMySQL8016AndMore(self):
+        version = float(self.getMySQLVersion().rsplit(".", 1)[0])
```
Comment (dongwook-chan) [inline pymysqlreplication/tests/base.py:88]: Any reasons for opting to cast to float instead of int?

Comment (mirageoasis) [reply pymysqlreplication/tests/base.py:88]: Because Other code converts version into float I really don't know
Can I investigate this problem?

Comment (dongwook-chan) [reply pymysqlreplication/tests/base.py:88]: Yes. I believe casting to `int` is enough.
We should apply Boy Scout Rule.
Please investigate.

Comment (dongwook-chan) [reply pymysqlreplication/tests/base.py:88]: My bad. I mistook the number of split tokens.

**Pull request merged**